### PR TITLE
docs: Move Post install link to infobox

### PR DESCRIPTION
**Pull request by KatrinaAS**
Moved Post-Install link to inbobox to make it more visible.

### Proposed changes

This was done as when people are trying to install Docker to use software I assist in maintain. I have countless times had to link them to the post-install instructions, to the point of one of the people maintain it has suggested linking to it in our own README.md. As they are so unclear to find unless you read what looks to be boilerplate "You have completed install" text

### CODE REVIEW

**Comment by KatrinaAS** [comment]
Added a title to the infobox to make it more clear.

File: engine/install/ubuntu.md [hunk 1]
```diff
@@ -180,11 +180,12 @@ Docker from the repository.
    This command downloads a test image and runs it in a container. When the
    container runs, it prints a confirmation message and exits.
 
-You have now successfully installed and started Docker Engine. The `docker` user
-group exists but contains no users, which is why you're required to use `sudo`
-to run Docker commands. Continue to [Linux post-install](linux-postinstall.md)
-to allow non-privileged users to run Docker commands and for other optional
-configuration steps.
+You have now successfully installed and started Docker Engine. 
+> Receiving errors when trying to run without root?
```
Comment (thaJeztah) [inline engine/install/ubuntu.md:184]: Could you keep an empty line between the line and the infobox?
```suggestion
You have now successfully installed and started Docker Engine.

> Receiving errors when trying to run without root?
```

**Review by thaJeztah — COMMENTED** [review]
If we change this page, we should probably change all the other Linux distros (such as the [`debian.md` in the same directory](https://github.com/docker/docs/blob/main/engine/install/debian.md) with the same changes

**Comment by KatrinaAS** [comment]
> If we change this page, we should probably change all the other Linux distros (such as the [`debian.md` in the same directory](https://github.com/docker/docs/blob/main/engine/install/debian.md) with the same changes

Definatly makes sense, will do that

**Comment by KatrinaAS** [comment]
I made sure to follow the existing documentation, so only mentioned the docker group existing if the existing documentation did.

**Comment by ChrisChinchilla** [comment]
Thanks @KatrinaAS I refactored the change into using an include so we can change the text in one place. For some reason my change isn't showing up here… But GitHub had some issues today, maybe it's taking it's time… 🤔

**Review by aevesdocker — APPROVED** [review]

**Comment by ChrisChinchilla** [comment]
Thanks @KatrinaAS !

**Pull request merged**